testing/src/cypress/e2e/02-institution/01-login-institution-page.cy.ts

```diff
@@ -1,22 +1,34 @@
 import DashboardInstitutionObject from "../../page-objects/Institution-objects/DashboardInstitutionObject";
 import InstitutionCustomCommand from "../../custom-command/institution/InstitutionCustomCommand";
+import LoginInstitutionObject from "../../page-objects/Institution-objects/LoginInstitutionObject";
 
-describe("Login Page", () => {
-  const dashboardInstitutionObject = new DashboardInstitutionObject();
-  const institutionCustomCommand = new InstitutionCustomCommand();
+const dashboardObject = new DashboardInstitutionObject();
+const iCC = new InstitutionCustomCommand();
```

Comment:
better to give meaningful variable names

Reply:
Agreed. Will revert it back. Wanted to keep the strings shorter.